functions-and-scope.js

```diff
@@ -43,6 +66,24 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // ---- Verwachte uitkomst: 6.642857142857143
 
 
+const gradesTemp = grades.slice(); //Creëer een referentieloze lijst op basis van de grades lijst
+
+let amountGrades = 0;
+
+//maak een for loop waarbij elk cijfer bij het vorige wordt opgeteld, om zo het totaal uit de lijst te krijgen
+function calculateAverage() {
+    amountGrades = gradesTemp.length; //Bereken aantal cijfers in cijfers
+    for (let i = 1; i < amountGrades; i++) {
+        gradesTemp[0] = gradesTemp[0] + gradesTemp[i]; // i waarde optellen bij item 0 uit lijst
+    }
+    const totalGrades = gradesTemp[0]; //geeft totaal item 0 uit lijst
+    const averageGrade = totalGrades / amountGrades; //deel het totaal van alle cijfers door het aantal cijfers in de lijst
+    return averageGrade;
```

Comment:
Dit zou ik ook anders doen. Gewoon het ophogen van een variabele in plaats van een waarde uit de array manipuleren.
```
let counter = 0;
for (let i = 0; i < gradesCumLaude.length; i++) {
    counter += gradesCumLaude[i];
}
return counter / gradesCumLaude.length;
```